.travis.yml

```diff
@@ -19,4 +19,4 @@ before_install:
   - tar -xzf ../apache-cassandra-2.1.12-bin.tar.gz -C ../
   - sh ../apache-cassandra-2.1.12/bin/cassandra > /dev/null
 
-script: sh test/utils/run_tests.sh coverage all && (npm run-script coveralls || exit 0)
+script: (sh test/utils/run_tests.sh coverage all && npm run-script coveralls || exit 0)
```

Comment:
Good point. 

I could swear that there have been several cases recently where the return status was non-zero despite all tests passing, but can't seem to find the output any more. I'll look more closely the next time this happens.